cluster/images/hyperkube/Dockerfile

```diff
@@ -14,5 +14,8 @@
 
 FROM BASEIMAGE
 
-# Copy the hyperkube binary
+# Cleanup all the soft links
+ADD binaries.tgz /usr/local/bin
+
+# Copy the shell script
 COPY hyperkube /hyperkube
```

Comment:
should we set the entrypoint ?? we are in the bazel image

Reply:
Done